backends/qualcomm/tests/test_qnn_delegate.py

```diff
@@ -121,22 +121,6 @@ def forward(self, x):
         model_name = "qnn_mean_dim_model"
         save_model_and_expected_output(instance, buffer, example_inputs, model_name)
 
-    def test_qnn_backend_addmm(self):
```

Comment:
why is this test removed?

Reply:
I think we don't really have an `addmm` operator in existing example models, and there also has no builder for it.
We will add it back once the use case was found.